modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Cases/CaseForm.tsx

```diff
@@ -111,7 +111,7 @@ const CaseForm = () => {
 					...testrayCase,
 					caseTypeId: testrayCase.caseType?.id,
 					componentId: testrayCase.component?.id,
-					priority: priorities[0].value,
+					priority: testrayCase.priority,
```

Comment:
Since in the line 111 we set `...testrayCase`, priority is already there.

So we could follow that way, or remove that line would work the same xD